### ISSUE TITLE
Reproducbility with Universal-Sentence-Encoder

### ISSUE THREAD

**Issue by iamdank** [MaartenGr/BERTopic#737]
I followed the [instructions](https://maartengr.github.io/BERTopic/faq.html#why-are-the-results-not-consistent-between-runs) to ensure consistent / reproducible results using sBERT transformers. This works fine.

However, I was unable to get reproducible results via the USE v4 embedding.
Attached is the code I used, any tips would be kindly appreciated.

`from bertopic import BERTopic
import tensorflow_hub
from umap import UMAP
from sklearn.datasets import fetch_20newsgroups

docs = fetch_20newsgroups(subset='all', remove=('headers', 'footers', 'quotes'), shuffle=True, random_state=42)['data']

umap_model = UMAP(n_neighbors=15, n_components=5, 
                  min_dist=0.0, metric='cosine', random_state=42)

embedding_model = tensorflow_hub.load("https://tfhub.dev/google/universal-sentence-encoder/4")
topic_model = BERTopic(embedding_model=embedding_model, umap_model=umap_model, verbose=True)

topics, probs = topic_model.fit_transform(docs)`

**Comment by MaartenGr** [comment]
I believe this is as a result of the embedding model that does not consistently produce the same embeddings but I am not entirely sure:

```python
from sentence_transformers import SentenceTransformer

# This works
sentence_model = SentenceTransformer("all-MiniLM-L6-v2")
embeddings = sentence_model.encode(docs[:10], show_progress_bar=False)
embeddings1 = sentence_model.encode(docs[:10], show_progress_bar=False)
assert np.array_equal(embeddings, embeddings1)

# This does not work
embedding_model = tensorflow_hub.load("https://tfhub.dev/google/universal-sentence-encoder/4")
embeddings = np.array([embedding_model([doc]).cpu().numpy()[0] for doc in docs[:10]])
embeddings1 = np.array([embedding_model([doc]).cpu().numpy()[0] for doc in docs[:10]])
assert np.array_equal(embeddings, embeddings1)
```
